api/monetization.js

```
@@ -0,0 +1,277 @@
+
+  try {
+    switch (event.type) {
+      case 'checkout.session.completed': {
+        const session = event.data.object;
+        const tier = session.metadata?.tier || 'PRO';
+        const email = session.customer_email;
+        const customerId = session.customer;
+        const subscriptionId = session.subscription;
+
+        // Generate license token
+        const token = createLicenseToken(customerId, email, tier, 400);
```

ESLint

[error] 'token' is assigned a value but never used.

(no-unused-vars)